src/release/tests.rs

```diff
@@ -221,6 +216,56 @@ fn test_try_merge_v2() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+#[test]
+fn test_try_merge_v1() -> Result<(), Box<dyn Error>> {
+    // Load a v1 META file.
+    let dir: PathBuf = [env!("CARGO_MANIFEST_DIR"), "corpus"].iter().collect();
+    let widget_file = dir.join("v1").join("widget.json");
+    let mut contents: Value = serde_json::from_reader(File::open(&widget_file)?)?;
+
+    // Insert release metadata.
+    let obj = contents.as_object_mut().unwrap();
+    obj.insert("user".to_string(), json!("omar"));
+    obj.insert("date".to_string(), json!("2023-07-23T08:54:32.386"));
+    obj.insert(
+        "sha1".to_string(),
+        json!("ca8716f3b0c65ec10207acbe93e09dadbecfbf92"),
+    );
+
+    // expect maps a JSON pointer to an expected value.
+    for (name, patches, expect) in [
+        (
+            "license",
+            vec![json!({"license": "MIT"})],
```

Comment:
```suggestion
            &[json!({"license": "MIT"})],
```

Doing this I believe you could skip the `.as_slice()` below

Reply:
Sadly that doesn't work because the third item in the list has two items in the vector, not one, so it complains with

```
mismatched types
expected reference `&[serde_json::Value; 1]`
   found reference `&[serde_json::Value; 2]`
```